batchthresholdbundling.php

```diff
@@ -75,7 +92,8 @@ function batchthresholdbundling_civicrm_batchItems(&$queryResults, &$financialIt
       // $unsetIDs store entries IDs that are bundled up into one and need to be deleted
       $totalAmounts = $unsetIDs = [];
       foreach ($accountCollection as $id => $account) {
-        if ($entries[$id]['CONTRIBUTION_AMOUNT'] <= $thresholdAmount) {
+        if ($entries[$id]['CONTRIBUTION_AMOUNT'] <= $thresholdAmount ||
+            ($bundleDebits && (int) $entries[$id]['AMOUNT'] > 0)) {
```

Comment:
I'm not quite sure that this will work properly in a general case for cancelled grant payments nor for payments or cancellations of payments 0.01 <= AMOUNT <= 0.49 (rare but possible edge case).

Reply:
@JoeMurray I forgot about this, but you were absolutely correct.  We picked this up when they added a Stripe account, since the fee transactions could come in at under 50 cents.